multicluster/charts/linkerd-multicluster/templates/gateway.yaml

```diff
@@ -29,10 +34,32 @@ spec:
       labels:
         app: {{.Values.gateway.name}}
     spec:
+      {{- if .Values.enablePodAntiAffinity -}}
+      {{- $local := dict "component" "gateway" -}}
+      {{- include "linkerd.pod-affinity" $local | nindent 6 -}}
+      {{- end }}
       containers:
         - name: pause
           image: gcr.io/google_containers/pause
       serviceAccountName: {{.Values.gateway.name}}
+{{- if .Values.enablePodAntiAffinity }}
+---
+kind: PodDisruptionBudget
+apiVersion: policy/v1beta1
+metadata:
+  name: linkerd-dst
```

Comment:
This should be named `{{.Values.gateway.name}}`

Reply:
That it should. 👍